src/components/loading-empty-results/index.tsx

```diff
@@ -8,11 +8,13 @@ import style from './loading-empty-results.module.css';
 
 export type LoadingEmptyResultsProps = PropsWithChildren<{
   isSearching: boolean;
+  isGeolocationError?: boolean;
   type: keyof typeof ComponentText.EmptySearch.searching;
 }>;
 
 export default function LoadingEmptyResults({
   isSearching,
+  isGeolocationError = false,
```

Comment:
Wonder if these could be union types instead 🤔 Instead of having several semi-contradictory booleans. But that would maybe move the logic responsibility to the consumer 🤔

Reply:
Yeah, they are sort of mutually exclusive. At least in the output of the component.

Not sure which solution is better though. Making it a union type moves the responsibility over to the consumer as you say, and might be more complex when ternaries are used to set the property based on isSearching or isGeolocationError.

Reply:
lets not solve it now in any case